setup.py

```diff
@@ -89,12 +89,16 @@ def absolute_links(txt):
             # 'pypesto @ git+https://github.com/ICB-DCM/pyPESTO.git@develop#egg=pypesto',
         ],
         'doc': [
-            'sphinx>=3.5.3',
+            'sphinx>=3.5.3,<7',
```

Comment:
Can this `<7` be removed when the lower `# pin` comment is removed?

If so then
```suggestion
            # remove <7 when below nbsphinx==0.9.1 and nbconvert<7.5.0 are unpinned
            'sphinx>=3.5.3,<7',
```

Reply:
Honestly, I don't know. It's usually trial-and-error with the documentation stack... :see_no_evil: